pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/ModularLoadManagerImplTest.java

```diff
@@ -630,4 +630,14 @@ public void testOwnBrokerZnodeByMultipleBroker() throws Exception {
 
         pulsar.close();
     }
+
+    @Test
+    public void testZnodeMissed() throws Exception {
+        String path = LoadManager.LOADBALANCE_BROKERS_ROOT + "/" + pulsar1.getAdvertisedAddress() + ":" + pulsar1.getConfiguration().getWebServicePort().get();
+        ZkUtils.deleteFullPathOptimistic(pulsar1.getZkClient(), path, -1);
+        // Wait broker write load balance data to zookeeper, default interval is 5s.
+        Thread.sleep(6000);
```

Comment:
Can we avoid sleeping in the test? It introduces flakiness to the whole test suite. for example manually trigger load manager to write load balance data to zookeeper.

Reply:
Changed to manually trigger load manager to write load balance data to zookeeper